frontend/src/metabase/visualizations/components/TableInteractive.jsx

```diff
@@ -438,8 +443,9 @@ export default class TableInteractive extends Component {
     const isClickable = this.visualizationIsClickable(clicked);
     const backgroundColor = this.getCellBackgroundColor(
       settings,
+      value,
       rowIndex,
-      columnIndex,
+      column.name,
```

Comment:
Why do we now need value and name rather than index? I guess that change happened before and broke formatting?

Reply:
I think I broke it when I refactored some things. The conditional formatting settings always used column name as the key rather than index.